### PR TITLE
[Test Gap] Add a test script to cover show ip bgp nettwork

### PR DESCRIPTION
**Pull request by ZhaohuiS**
### Description of PR

Summary:
Fixes # (issue)
Address test gap https://github.com/sonic-net/sonic-mgmt/issues/5069
### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311
- [x] 202405

#### What is the motivation for this PR?
To address test gap https://github.com/sonic-net/sonic-mgmt/issues/5069
#### How did you do it?

1. Compare the output of show ip bgp network and docker exec -i bgp vtysh -c "show bgp ipv4 all"
2. Compare the routes and total path value is equal or not for both command
3. Both cover ipv4 and ipv6 command.

#### How did you verify/test it?
Run `bgp/test_bgp_command.py`

#### Any platform specific information?
All platforms

#### Supported testbed topology if it's a new test case?
All topologies

### CODE REVIEW

**Comment by StormLiangMS** [comment]
hi @prsunny could you help to review?

File: tests/bgp/test_bgp_command.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    bgp_docker_output = duthost.shell(bgp_docker_cmd)["stdout"]
+    pytest_assert(
+        "*=" in bgp_docker_output,
+        "Failed to run '{}' command".format(bgp_docker_output),
+    )
+    # Remove the first two lines from the docker command output
+    bgp_docker_output_lines = bgp_docker_output.splitlines()[2:]
+    bgp_docker_output_modified = "\n".join(bgp_docker_output_lines)
+
+    pytest_assert(
```
Comment (prsunny) [inline tests/bgp/test_bgp_command.py:56]: It is possible that there could be cases where bgp docker is down on the DUT. Do we need to consider that as a test failure for this scenario? Meaning, if our intention is to check the correctness b/w the two commands, we can fail the test only if docker output is successful and network output failed.  If both failed, then it's not related to this test. What do you think?

Comment (ZhaohuiS) [reply tests/bgp/test_bgp_command.py:56]: @prsunny If bgp docker is down, there will be no "*=" in the output, and case will fail. I can also add a check to see if rc equals 0. Yes, we will consider it as case failure, sanity check passes but here bgp docker container is down, that could be some problem and case failure can cause our attention.
My intention is to capture not only the output of show ip bgp network, but also check the bgp container health.

Comment (prsunny) [reply tests/bgp/test_bgp_command.py:73]: ok

**Review by prsunny — APPROVED** [review]
lgtm

**Comment by StormLiangMS** [comment]
new case, let's hold till it is approved stable to backport.

**Pull request merged**